### PR TITLE
swagger docs

### PR DESCRIPTION
**Pull request by JenMcD-star**
currently runs on the local host (http://localhost:5005/api-docs/). I will update if/when we deploy the app. Having trouble linking the yaml file in the app.js without using the full path. Even the relative path is returning as doesn't exist. Any ideas?

### CODE REVIEW

File: src/swagger.yaml
Comment (jamisonvalentine) [inline src/swagger.yaml:1]: What's the motivation behind the extra whitespace?

**Comment by jamisonvalentine** [comment]
> currently runs on the local host (http://localhost:5005/api-docs/). I will update if/when we deploy the app. Having trouble linking the yaml file in the app.js without using the full path. Even the relative path is returning as doesn't exist. Any ideas?

I think [`path.relative`](https://nodejs.org/api/path.html#pathrelativefrom-to) or [`path.dirname`](https://nodejs.org/api/path.html#pathdirnamepath) would be great options here

File: src/swagger.yaml
Comment (JenMcD-star) [reply src/swagger.yaml:1]: So, the way code the dream showed us how to use swagger is to export docs from postman, import them to apimatic and that generates the yaml. Part of what it generates is the whitespace. If I take out the whitespace then all of the lines need to also be moved and it's very picky about indentations so I end up with error messages everywhere. If there's a better way to do it I'm totally open to it! I did ask my reviewer in the final project if I did it correctly and he basically said the docs were working so it was fine.

**Review by Count-MonteCristo — APPROVED** [review]

File: src/swagger.yaml
Comment (jamisonvalentine) [reply src/swagger.yaml:1]: Fair enough!

**Review by jamisonvalentine — APPROVED** [review]

**Pull request merged**